app-modules/panel-organization/src/Filament/Resources/Recruitment/Applications/Schemas/ApplicationInfolist.php

```
@@ -62,10 +65,18 @@ public static function configure(Schema $schema): Schema
                             ]),
 
                         Tab::make('Comments')
+                            ->id('comments')
                             ->label(__('panel-organization::filament.tabs.comments'))
                             ->schema([
-                                ViewEntry::make('comments')
-                                    ->view('panel-organization::components.applications.tabs.comments'),
+                                CommentsEntry::make('comments')
+                                    ->mentionables(fn (Application $record) => once(
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$record'. (undefined)

(UnusedFormalParameter)